tests/integration/ceph_base_nfs_test.go

```diff
@@ -29,6 +29,8 @@ import (
 // Smoke Test for File System Storage for CephNFS - Test check the following operations on Filesystem Storage in order
 // Create,Mount,Write,Read,Unmount and Delete.
 func runNFSFileE2ETest(helper *clients.TestClient, k8sh *utils.K8sHelper, s suite.Suite, settings *installer.TestCephSettings, filesystemName string) {
+	// TODO: remove Skip() once issue https://github.com/rook/rook/issues/10518 is resolved.
+	s.T().Skip("skipping nfs e2e to avoid flakiness")
```

Comment:
What do we know about the flakiness? If it's later in the test, could this skip move later in the test so we at least get a partial test until the issue is fixed?

Reply:
https://github.com/rook/rook/issues/10518#issuecomment-1169072008 

nfs pvc creation / deletion fails at times, 
Modified to skip only nfs csi tests now.
PTAL